src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +54,34 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService = (MovieSessionService)
+                injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        AuthenticationService authenticationService = (AuthenticationService)
+                injector.getInstance(AuthenticationService.class);
+
+        User exampleUser = null;
+        try {
+            exampleUser = authenticationService.register("user@gmail.com", "123455667");
+        } catch (RegistrationException e) {
+            System.out.println("Can`t register user: " + e);
+        }
```

Comment:
Consider providing more informative feedback when catching `RegistrationException`. This will help in debugging and understanding the issue better.